src/core/server/saved_objects/service/lib/search_dsl/query_params.ts

```diff
@@ -39,17 +39,21 @@ function getTypes(mappings: IndexMapping, type?: string | string[]) {
 }
 
 /**
- *  Get the field params based on the types and searchFields
+ *  Get the field params based on the types, searchFields, and rawSearchFields
  */
-function getFieldsForTypes(types: string[], searchFields?: string[]) {
-  if (!searchFields || !searchFields.length) {
+function getFieldsForTypes(
+  types: string[],
+  searchFields: string[] = [],
+  rawSearchFields: string[] = []
```

Comment:
Adding `rawSearchFields` seemed to be the least disruptive thing I could do to achieve the kind of querying that I need during import pre-flight checks.

`searchFields` are assumed to be for attributes of the given saved object type, so they always get prefixed with `${type}.` when converted to the search DSL. `rawSearchFields`, on the other hand, allow other fields (such as `_id` and `originId`) to be queried without that prefix.